src/app/home/home.component.ts

```diff
@@ -72,15 +72,11 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
         public darkModeService: DarkModeService
     ) {
         this.initializeURLs();
-        // should any other call be made here? probably not... actually even this should be removed soon
-        this.http.get(this.pingURL).subscribe(
-            () => {
-                console.log('pinging server...');
-            },
-            () => {
-                this.disableEditor();
-            }
-        );
+
+        this.http.get(this.pingURL).subscribe({
+            next: () => console.log('pinging server...'),
+            error: () => this.disableEditor()
+        });
```

Comment:
FYI @Kledis98 this has been slightly changed, have a look at it (from what I can tell, the old variant was deprecated, see why the observer object seems to be preferred).